server/server.go

```diff
@@ -129,6 +129,7 @@ func (s *server) ServeFile(ctx context.Context, w http.ResponseWriter, r *http.R
 	}
 
 	commit := r.URL.Query().Get("commit")
+	ffl := r.URL.Query().Get("ffl")
```

Comment:
Can this logic, and the big `if` statement below, now go away since `ffl=...` should be handled entirely in the controller and never be visible to the JavaScript?